content/en/synthetics/api_tests/grpc_tests.md

```diff
@@ -0,0 +1,186 @@
+
+### Specify test frequency
+
+gRPC tests can run:
+
+* **On a schedule** to ensure your most important endpoints are always accessible to your users. Select the frequency at which you want Datadog to run your gRPC test.
+* [**Within your CI/CD pipelines**][4] to start shipping without fearing faulty code might impact your customers experience.
+* **On-demand** to run your tests whenever makes the most sense for your team.
+
+### Define alert conditions
+
+Set alert conditions to determine the circumstances under which you want a test to fail and trigger an alert.
+
+#### Alerting rule
+
+When you set the alert conditions to: `An alert is triggered if any assertion fails for X minutes from any n of N locations`, an alert is triggered only if these two conditions are true:
+
+* At least one location was in failure (at least one assertion failed) during the last *X* minutes;
+* At one moment during the last *X* minutes, at least *n* locations were in failure.
+
+#### Fast retry
+
+Your test can trigger retries `X` times after `Y` ms in case of a failed test result. Customize the retry interval to suit your alerting sensibility.
+
+Location uptime is computed on a per-evaluation basis (whether the last test result before evaluation was up or down). The total uptime is computed based on the configured alert conditions. Notifications sent are based on the total uptime.
+
+### Notify your team
+
+A notification is sent by your test based on the [alerting conditions](#define-alert-conditions) previously defined. Use this section to define how and what message to send to your teams.
+
+1. [Similar to how you configure monitors][6], select **users and/or services** that should receive notifications either by adding a `@notification`to the message or by searching for team members and connected integrations with the drop-down box.
+
+2. Enter the notification **message** for your test. This field allows standard [Markdown formatting][7] and supports the following [conditional variables][8]:
+
+    | Conditional Variable       | Description                                                         |
+    |----------------------------|---------------------------------------------------------------------|
+    | `{{#is_alert}}`            | Show when the test alerts.                                          |
+    | `{{^is_alert}}`            | Show unless the test alerts.                                        |
+    | `{{#is_recovery}}`         | Show when the test recovers from alert.                             |
+    | `{{^is_recovery}}`         | Show unless the test recovers from alert.                           |
+
+3. Specify how often you want your test to **re-send the notification message** in case of test failure. To prevent renotification on failing tests, leave the option as `Never renotify if the monitor has not been resolved`.
```

Comment:
```suggestion
3. Specify how often you want your health check test to **re-send the notification message** in case of test failure. To prevent renotification on failing health check tests, leave the option as `Never renotify if the monitor has not been resolved`.
```